inc/export/cron/namespace.php

```diff
@@ -0,0 +1,229 @@
+<?php
+/**
+ * Altis Analytics Data Export Cron Job.
+ */
+
+namespace Altis\Analytics\Export\Cron;
+
+use Altis\Analytics\Utils;
+use Aws\S3\Exception\S3Exception;
+use Aws\S3\S3Client;
+
+/**
+ * Frequency of the export job
+ */
+const ALTIS_ANALYTICS_EXPORT_CRON_FREQUENCY = 'ten_minutes';
+const ALTIS_ANALYTICS_EXPORT_CRON_FREQUENCY_INTERVAL = 600;
+
+/**
+ * Option name to keep last processed file information.
+ */
+const ALTIS_ANALYTICS_EXPORT_LAST_PROCESSED_KEY = 'altis.analytic.export.last_processed_key';
+
+/**
+ * Bootstrap the functionality.
+ *
+ * @return void
+ */
+function bootstrap() : void {
+
+	/**
+	 * Filter to allow disabling the functionality altogether.
+	 *
+	 * Defaults to work only on the main site of the network.
+	 *
+	 * @param bool Whether to enable the data export cron job.
+	 */
+	if ( ! apply_filters( 'altis.analytics.export.cron.enabled', is_main_site() ) ) {
```

Comment:
I was thinking this gives the room for developers to manage this granularly, in case there is a way to split analytics data by site or something. I do see the concern about incorrect use, I had that initially then opted to give the flexibility. Happy to go the safer route.